app/src/main/java/org/mozilla/focus/browser/integration/FullScreenIntegration.kt

```diff
@@ -87,32 +88,14 @@ class FullScreenIntegration(
      */
     @VisibleForTesting
     internal fun switchToImmersiveMode() {
-        val window = activity.window
-        window.addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
-        @Suppress("DEPRECATION") // https://github.com/mozilla-mobile/focus-android/issues/5016
-        window.decorView.systemUiVisibility = (
-            View.SYSTEM_UI_FLAG_LAYOUT_STABLE
-                or View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION
-                or View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN
-                or View.SYSTEM_UI_FLAG_HIDE_NAVIGATION
-                or View.SYSTEM_UI_FLAG_FULLSCREEN
-                or View.SYSTEM_UI_FLAG_IMMERSIVE_STICKY
-            )
+        activity.enterToImmersiveMode()
     }
 
     /**
      * Show the system bars again.
      */
     fun exitImmersiveModeIfNeeded() {
```

Comment:
The method name may need updating.
The `..IfNeeded` part is now a bit confusing. Maybe just `exitImmersiveMode` is enough.

Reply:
Removed the Ifneeded part